### ISSUE TITLE
route naming convention 

### ISSUE THREAD

**Issue by eugv86** [emiliogrv/nativescript-vue-router#4]
@emiliogrv Is there a particular reason for naming the routes "dashboard.index" with the .index?

**Comment by emiliogrv** [comment]
@eugv86 nope, there isn't, that's just a name that I like, you can use your own style to name them

**Comment by eugv86** [comment]
Thanks for the response. I'm just trying to figure out why this peace of code never matches for me `reduce(router.vue, '$options.name') === reduce(route, 'component.name')`
